juju/juju
# Use podman if it is installed. Fix DOCKER_USERNAME for podman.

- Changes to prefer podman over docker.
- Podman doesn't use the default domain of docker.io in oci image references. This prefixes our docker org with docker.io to be compatible with podman and docker.

## QA steps

With only docker installed
- `make operator-image` should use docker
With podman installed (with or without docker also installed)
- `make operator-image` should use podman
- Images created should be prefixed with docker.io by default.

## Documentation changes

N/A

## Links

N/A
